test/parallel/test-fs-existssync-false.js

```diff
@@ -1,12 +1,15 @@
 'use strict';
-
 const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+// This test ensures that fs.existsSync doesn't incorrectly return false
+// (especially on Windows)
```

Comment:
Nit: It would be nice to punctuate the new comments.

Reply:
Isn't that in conflict with the canonical test structure?

Ref: https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-1-3